semcore/feedback-form/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [6.29.4] - 2024-06-10
+
+### Fixed
+
+- i18n token in `FeedbackRating` Notice.
```

Comment:
Basically we fixed aria label here. Could you also add tralsnations?

Reply:
changed note and added translations